### DISCUSSION TITLE
新聞業開始應用人工智慧，但你相信 AI 寫的新聞嗎？

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2047] [General]
https://technews.tw/2024/07/15/do-you-trust-ai-generated-news-content/

因為不了解生成式AI的運作背景與方式，對AI生成的新聞自然會帶點懷疑，目前要辨識出是否為AI寫的新聞也有難度。
但我很認同文中提出的解決方法:讓AI成為人類記者的助手，利用大數據分析出熱門新聞主題，記者再運用專業知識、情感共鳴和價值判斷及批判思考，完成深入的報導討。
所以本質上，還是由記者本人藉由與AI的互動討論、下指令，引導出高品質的新聞主題，不會出現「AI取代人類記者」的問題。

其實我認為所有領域都能以此模式來運用AI，不但能減少人工耗時、提升成果產出的效率，也保留了人類獨有的情感、價值觀及原創性等等，或許也是目前人類與AI最適合的相處模式。
在生活中有那些與生成式AI合作得不錯的例子或經驗?
歡迎留言分享和推薦好用的功能~

**Comment by Penny-712** [comment]
我覺得AI寫新聞的好處是它不會帶有個人的價值觀或是政治色彩，會比較客觀的陳述事實。但AI畢竟沒辦法實地考察、親眼目睹事情的發生經過，所以寫出來的文字可能就沒有那麼有渲染力。如果是人類寫文章，AI幫忙校稿、抓錯等，互相合作，我認為是最好的互動方式。

**Comment by MingWei174** [comment]
AI生成的「單純陳述事實」的新聞內容對於我來說是稍顯不足的，但我也無法接受新聞媒體利用聳動、吸睛的「釣魚標題」騙取點擊率。
在各種新聞播報題材中，我更喜歡記者在報導的同時穿插記者在目睹一件事當下的「個人」見解以及各種真實的反應，又或者是分享自己回憶種發生過的故事、場景描述的專題報導，透過故事的鋪陳，能讓我更快地進入報導情境，並且增加報導的可讀性。

**Comment by Nana1218** [comment]
我覺得更應該在AI及人類寫報導中取得平衡，人類會用帶有強烈主觀意見的方式寫出報導，更因為現在的大數據趨勢會下許多偏頗的標題，然而AI雖能用客觀角度進行描述，並擁有大量數據庫，但也卻因此少了最重要的批判性思維，不應該是用單一方式寫報導，人類與AI相輔相成才能創造出最好的結果。

**Comment by JoannaLai909** [comment]
我覺得AI應該主要是拿來輔助寫新聞，它可以代替人工去收集數據，減少許多人力成本，但如果把它作為主要的工具不太合適，雖然說寫新聞不應該有絕對的主觀情感，這方面AI不毀有這樣的問題，但我平常看新聞的時候看到真人報導時流露出的主觀意見甚至會對某些事件有一些個人情緒會覺得很有趣，所以我認為可以將AI作為輔助工具但不能完全取代

**Comment by ChristinaK1011** [comment]
我覺得AI更適合拿來寫有標準數據的客觀調查新聞，例如：一年之中臺灣製造了多少噸的垃圾、一年當中有多少位新生兒出生等，但如果是像政治相關新聞，通常都會有資深媒體人出來說他們的一些觀察和推論，這就必須得是真人來寫新聞和說明，若怕資訊偏頗，可以同時找不同立場的來賓來做分析。

**Comment by ds0w0** [comment]
我認為AI作為新聞業的工具是十分恰當的，由於記者有規定的新聞產量，AI加速記者寫作的效率或許能減少過於偏頗或沒有社會利益的新聞，且AI做為資料蒐集的輔助會使新聞更有可信度。前陣子我看到三體的作者(沒記錯的話)參與了一款針對文字創作者AI的測試，他表示有了AI的協助，除了可以花更多心思創新內容，沒靈感時也能由AI提供更多思路，期待AI的成熟也能開發人類的不同可能。

**Comment by kingsophy** [comment]
我認為新聞業使用AI輔助可以幫助他們以更高的效率來製作新聞，在現今這個強調使用人工智慧的社會當中是十分合適的決定。但是不能全部交給AI完成，而是先讓AI製作一遍後，再經由人工逐字逐句審核，使整篇新聞的口吻及語氣更能接近人們日常生活中的樣子，而非是讓人感到是流水線大量生產出來的僵硬感，這樣才能達到雙贏，就是所謂的業者營，閱聽者也贏。

**Comment by hunteryang0108** [comment]
新聞本身代表著有一個事件的存在，記者的工作是以文字或多媒體為載體傳播這個事件的相關描述，我認為這個工作是可以被AI所取代的，只要AI撰寫的文本內容背後有一個真實的事件存在，那這個文本內容就是值得信賴的。

**Comment by Jerryguo888** [comment]
生成式AI已在多領域有和人類合作，例如內容創作，AI協助生成初稿，創作者再優化；醫療診斷中，AI分析數據，醫生作出最終判斷；設計領域，AI生成草圖，設計師精修創意。
